packages/relay/src/lib/eth.ts

```diff
@@ -124,16 +126,22 @@ export class EthImpl implements Eth {
   }
 
   private async getFeeWeibars() {
-    const exchangeRates = await this.sdkClient.getExchangeRate();
-
-    //FIXME retrieve fee from fee API when released
-    const contractTransactionGas = 853454;
-
-    //contractTransactionGas is in tinycents * 1000, so the final multiplier is truncated by 3 zeroes for
-    // the conversion to weibars
-    return Math.ceil(
-      (contractTransactionGas / exchangeRates.currentRate.cents) * exchangeRates.currentRate.hbars * 10_000_000
-    );
+    const networkFees = await this.mirrorNodeClient.getNetworkFees();
+
+    if (networkFees && Array.isArray(networkFees.fees)) {
+      const txFee = networkFees.fees.find(({transaction_type}) => transaction_type === "EthereumTransaction");
```

Comment:
Make `EthereumTransaction` a static field.
Or add to constants if it is used elsewhere